packages/sw-lib/src/lib/router.js

```diff
@@ -38,46 +38,23 @@ import ErrorFactory from './error-factory.js';
  *   // The FetchEvent to handle
  *   console.log(args.event);
  *
- *   // The parameters from the matching route.
+ *   // The parameters from the matching route (Commonly
+ *   // used with Regex / Express routes).
  *   console.log(args.params);
  *
  *   // Return a promise that resolves with a Response.
  *   return fetch(args.url);
  * }));
  *
- * @example <caption>How to define a route using a Route instance.</caption>
- *
- * const routeInstance = new goog.swlib.Route({
- *   match: (url) => {
- *     // Return true or false
- *     return true;
- *   },
- *   handler: {
- *     handle: (args) => {
- *       // The requested URL
- *       console.log(args.url);
- *
- *       // The FetchEvent to handle
- *       console.log(args.event);
- *
- *       // The parameters from the matching route.
- *       console.log(args.params);
- *
- *       // Return a promise that resolves with a Response.
- *       return fetch(args.url);
- *     },
- *   },
- * });
- * self.goog.swlib.router.registerRoute(routeInstance);
- *
  * @memberof module:sw-lib
  */
-class RouterWrapper {
+class Router {
   /**
-   * Constructs a new RouterWrapper.
+   * An instance of this call can be accessed via `goog.swlib.router`. You
+   * should not instantiate this class yourself.
    */
   constructor() {
-    this._router = new Router();
+    this._router = new SWRoutingRouter();
```

Comment:
Just checking: developers will never interact with `SWRoutingRouter`, right? It's just used internally as an alias so that it doesn't conflict with `Router`?

Because that's a confusing name to use for something that would be developer-facing.